### ISSUE TITLE
Evaluate Licensing

### ISSUE THREAD

**Issue by HalosGhost** [PDF-Accessibility-Initiative/grand-plans#2]
Our goal should be to be as widely-compatible as possible. Though Copy-Left licenses are commendable (and preferable, in my opinion), having one may hinder adoption of the use of our library. That may be an acceptable trade-off, but we should consider the cost-benefits.

For some of the same reasons mentioned in [this document](https://www.gnu.org/licenses/why-not-lgpl.html), the GNU LGPL may be a great candidate.

**Comment by meskarune** [comment]
I am kind of leaning toward MIT license just to encourage both proprietary and free software to adopt this. With LGPL a lot of closed source software won't want to use it based off the political implications alone >.> Even if it wouldn't affect their own software licensing. However it would be pretty shitty if some company made improvements or edits and then never released them so idk.

**Comment by HalosGhost** [comment]
Yeah; I understand that too. I suppose we could explicitly dual license:

* Offer it as LGPL for everyone
* Offer companies the option to pay us for a different license, probably MIT

Thoughts?

**Comment by meskarune** [comment]
Oooh,that's an interesting idea. It would force companies to contribute *something* in return for a less restrictive license.

One of the biggest issues with accessibility in software is how companies will charge large fees for it. In places like schools, accessibility is required by law. Many times insurance will cover some software packages too, and then companies over charge because they can. In the end people take advantage of those with disabilities who need accessible software to do anything with a computer, and then make a lot of money off them. If its possible to reverse that trend I would be for it.
